### PR TITLE
Release/1.1.1

### PR DESCRIPTION
**Pull request by zionts**

### CODE REVIEW

File: source/proxy-config.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The Engine Proxy has many configuration options. While all you need to set to ge
 
 If you are using the `apollo-engine` npm package, the parameters described here are the options to the `new ApolloEngine` and `new ApolloEngineLauncher` constructors. If you are using the Docker container distribution, the parameters described here go into your `engine-config.json` file.
 
-The `ApolloEngine` class in the `apollo-engine` npm package takes care of filling in a few important parameters for you automatically: `frontends.host`, `frontends.port`, `frontends.endpoints`, `origins.http.url`, and `origins.http.useFrontendPath`. If you are using that class, all you need is your `apiKey` to get started.
+The `ApolloEngine` class in the `apollo-engine` npm package takes care of filling in a few important parameters for you automatically: `frontends.host`, `frontends.port`, `frontends.endpoints`, `origins.http.url`, and `origins.http.useFrontendPath`. If you are using that class, all you need is your `apiKey` to get started (which can also be set with the `ENGINE_API_KEY` environment variable).
```
Comment (zionts) [inline source/proxy-config.md]: This parenthetical somehow got deleted. I've added it back in to `proto-doc-template.md` in monorepo, so this won't happen again.

File: scripts/version.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* global hexo */
 
 hexo.extend.tag.register('proxyDockerVersion', function(args) {
-  return '1.1';
+  return '1.1.1';
```
Comment (glasser) [inline scripts/version.js]: Not needed (docs about this are explicit if perhaps confusing) — I think it makes sense to have the recommended docker tag be the "latest of this minor version" one. So people will get patches when they re-pull their tag, but not minor/major changes.

File: source/proxy-release-notes.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ title: Proxy release notes
 
 The versions given here are both for the [`apollo-engine` Node.js package](https://www.npmjs.com/package/apollo-engine) and the `gcr.io/mdg-public/engine` Docker container.
 
+<h2 id="v1.1.1" title="v1.1.1">1.1.1 - 2018-05-07</h2>
+
+* The Engine Proxy now cleans HTTP headers / variables with invalid UTF-8 so that they can still be reported to the cloud service
```
Comment (glasser) [inline source/proxy-release-notes.md]: Missing trailing period.  Slash could just be "and". Maybe more like:

The Engine Proxy now sanitizes invalid UTF-8 in HTTP headers and variables, fixing the error `Error reporting traces. error="POST https://engine-report.apollodata.com/api/ss/traces giving up after 6 attempts"`.

maybe?

File: source/proxy-release-notes.md [hunk 4]
```diff
@@ -4,6 +4,13 @@ title: Proxy release notes
...
+* You may now use Engine with named pipes on Windows machines, which will support Node server deployments to Microsoft Azure. Instead of using the `port` argument in `engine.listen(port: process.env.PORT, httpServer: foo)`, you can now specify the `pipePath` argument to listen on a named pipe such as `engine.listen(pipePath: “\\.\pipe\bar”, httpServer:foo)`. In Microsoft Azure, `process.env.PORT`will be an acceptable input to `pipePath`.
```
Comment (glasser) [inline source/proxy-release-notes.md]: maybe "which will support" -> "to support"?

Both `listen` calls are missing their `{}`.

Missing space before second `foo`.

It appears that the quotes around the pipe path are "smart" quotes not normal ASCII double quotes.

Missing space before "will be an".

File: source/proxy-release-notes.md [hunk 5]
```diff
@@ -4,6 +4,13 @@ title: Proxy release notes
...
+* The Engine Proxy now differentiates request timeouts from failed requests. This will remove the “Unable to communicate with backend” error and replace it with two errors: one for no response or refused connection, and one for request timeouts.
+* The Engine Proxy now sets the `X-Forwarded-For` header appropriately and does string appending to other `X-Forwarded-` headers if they are already set.
```
Comment (glasser) [inline source/proxy-release-notes.md]: Maybe remove "appropriately" since that makes it seem like we used to do it inappropriately rather than not at all.

**Pull request merged**